shell/shared/demo.js

```diff
@@ -162,3 +176,34 @@ Router.map(function () {
     }
   });
 });
+
+Router.map(function () {
+  this.route("appdemo", {
+    path: "/appdemo/:packageId",
+    waitOn: function () {
+      return Meteor.subscribe("packageInfo", this.params.packageId);
+    },
+    onAfterAction: function() {
+      /* Stash the packageId into the Meteor session, so that, after
```

Comment:
Please use line (//) comments rather than block (/**/) comments. (Part of the style guide soon to be written! ;) )